### ISSUE TITLE
Content render problem after zooming

### ISSUE THREAD

**Issue by Pishtaz** [JoanZapata/android-pdfview#183]
Thank you for this job , i have problem when after zooming . see blow screenshot .

![screenshot_2016-02-28-22-59-37](https://cloud.githubusercontent.com/assets/4469948/13381343/0a159e3a-de6f-11e5-86af-c5e691f7e443.png)

any way to solve it ?

**Comment by Pommes9485** [comment]
Yeah, a many have. But so far, I did not find anything, to fix it. Really sad, because this is the only solution I found, which is free for commercial use. But like this, it is not really usable.

**Comment by SierraII** [comment]
Also having this issue with the latest version :(

**Comment by Pommes9485** [comment]
So, After hours of trying to compile it from github, I finally managed to import the pdf viewer by simply downloading and inserting the source. After adding the library I made it working. Although the issue here was not fixed, I simply inserted the merge requests and was able to fix it. The rendering process is still a bit slower than that from Adobe, but you gotta live with it. 

This is a very dirty solution, but I just do not know how to establish it in any other way.

**Comment by rrshaban** [comment]
@Pommes9485 could you point me towards the merge requests to the library that you're talking about? I've also been unable to find the 1.0.5 on Github (on the Releases page?).

**Comment by korujzade** [comment]
Same problem :(

**Comment by SierraII** [comment]
The only way to get this working is to follow Pommes9485 solution. Yes it is messy. No there is currently no other way around it.

Download the source code. Add it as a module to your Android App and change the grid size as shown in this commit.

[https://github.com/JoanZapata/android-pdfview/pull/175](url)

**Comment by korujzade** [comment]
@Pommes9485 @SierraII thanks for the solution. It works fine now. Do you guys know how can I hide rendering process when I zoom it so that users won't notice the rendering process?

**Comment by maxsays** [comment]
@korujzade Did the get the issue resolved regarding the rendering process? I am having the same one now after importing it as Module

**Comment by korujzade** [comment]
@maxsays yes. You also need to change GRID_SIZE variable from 7 to 10 in constant.java file as Sierrall has mentioned. 

https://github.com/JoanZapata/android-pdfview/pull/175/commits

**Comment by maxsays** [comment]
@korujzade I have got that solved already I was asking about hiding the rendering process? did you succeed in hiding it while it renders the page when we zoom in or out and pixels moves all over the screen.

**Comment by rrshaban** [comment]
@maxsays I've managed to minimize that by changing some of the other parameters in [Constants.java](https://github.com/dbchudasama/android-pdfview/blob/patch-2/android-pdfview/src/main/java/com/joanzapata/pdfview/util/Constants.java#L21), namely `PART_SIZE`, the dimension of the boxes, and `THUMBNAIL_RATIO`, the quality of the preview renderings. 

I've also experimented with increasing [`LOADED_SIZE`](https://github.com/JoanZapata/android-pdfview/blob/837d4f8723489723c7d73c7d2dd20bde3fdc3e17/android-pdfview/src/main/java/com/joanzapata/pdfview/PDFView.java#L492), but I think other changes would need to be made to take advantage of it.

**Comment by shashank8293** [comment]
@rrshaban Can you please mention the exact parameter changes that you made to minimize the rendering? It would be a great help. Thanks.
